src/data/download_raw.py

```diff
@@ -10,8 +10,11 @@
 data_raw_dir.mkdir(parents=True, exist_ok=True)
 
```

Comment:
My original purpose of this script was similar with make_dataset.py except make_dataset.py changes the batch size to 100,000. We could delete this file.